src/pheval/utils/phenopacket_utils.py

```diff
@@ -109,6 +109,15 @@ def observed_phenotypic_features(self) -> list[PhenotypicFeature]:
             phenotypic_features.append(p)
         return phenotypic_features
 
+    def negated_phenotypic_features(self) -> [PhenotypicFeature] or None:
```

Comment:
Is this a friendly thing to do in Python? This would hurt in Java where an empty list is preferred.

Reply:
Changed it!